app/controllers/credentials_controller.rb

```diff
@@ -29,16 +29,19 @@ def callback
       )
 
       if credential.update(
-        nickname: params[:credential_nickname],
+        nickname: credential_params[:nickname],
         public_key: webauthn_credential.public_key,
         sign_count: webauthn_credential.sign_count
       )
-        render json: { status: "ok" }, status: :ok
+        flash[:notice] = "Security Key registered successfully"
+        redirect_to root_path
       else
-        render json: "Couldn't add your Security Key", status: :unprocessable_content
+        flash[:alert] = "Couldn't register your Security Key"
+        render "home/index", status: :unprocessable_content
       end
     rescue WebAuthn::Error => e
-      render json: "Verification failed: #{e.message}", status: :unprocessable_content
+      flash[:alert] = "Verification failed: #{e.message}"
+      render "home/index", status: :unprocessable_content
```

Comment:
I don't really like this pattern to be honest... 🤔 

If something goes wrong when adding a passkey, the index page will be rendered but the `url` is `/credentials` so reloading the page will show the typical `Confirm form resubmission` alert

Not the end of the world but I wonder if there are other options here